metadata/manage.ts

```diff
@@ -1,5 +1,8 @@
 import Metadata, { Expression } from "./interface";
 
+export const ID_METADATA = "dsm-metadata";
+export const ID_METADATA_FOLDER = "dsm-metadata-folder";
+
```

Comment:
Love the refactoring here so everyone can access the constants, + using this to get rid of some magic strings.